recipes/yaqd-ti/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "yaqd-ti" %}
+{% set version = "2020.8.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/yaqd-ti-{{ version }}.tar.gz
+  sha256: 135d76cd2e167bad6a33711702c644d0c5bc32cefcb6a3e1573e218e98c12ec3
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - yaqd-ads1115 = yaqd_ti._ads1115:ADS1115.main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - flit
+    - yaqd-core >=2020.06.3
```

Comment:
Unless `yaqd-core` and `smbus2` are actually required at build-time (they don't appear to be, based on the `pyproject.toml`) then they don't need to be in `host:`

Reply:
@dopplershift flit (our build system) requires all dependencies at build time

Reply:
Alright, good to know.

Reply:
:thinking: hmm, that's a new one to me

here's my understanding: flit needs to install the package enough to import it, because flit extracts the `__version__` and package-level `__doc__` by import

I think that gidgethub gets away with not having dependencies at install time because the submodules are lazy-imported---that is, you interact with them using `from gidgethub import x`. So `import gidgethub` itself actually doesn't need any of the dependencies.

`yaqd-ti`'s [`__init__.py`](https://gitlab.com/yaq/yaqd-ti/-/blob/master/yaqd_ti/__init__.py) explicitly imports everything, so `import yaqd_ti` will fail if `yaqd-core` and `smbus2` are missing... :confounded: 

I will note that [all of the other yaq feedstocks](https://github.com/search?q=org%3Aconda-forge+yaq&unscoped_q=yaq) do include dependencies at build time, so if that's a problem it's one that we have to correct in many places.

I maintain upstream, so I _can_ change this behavior by modifying `yaqd-ti`'s `__init__`, what do you think?

Reply:
Well, I was trying to see how other packages do it. Clearly, there's a decently wide gamut of how things using flit are packaged on conda-forge. I think we can just merge.